sources/PTYSession.m

```diff
@@ -2360,6 +2364,7 @@ - (void)writeTaskNoBroadcast:(NSString *)string
 
 - (void)handleKeyPressInCopyMode:(NSEvent *)event {
     [self.textview setNeedsDisplayOnLine:_copyModeState.coord.y];
+    [self.textview setNeedsDisplayOnCol:_copyModeState.coord.x];
```

Comment:
Would be better to only do `setNeedsDisplayOnColumn:` when the vertical guide is enabled because we can't draw less than a whole row at a time (so drawing a column means redrawing the whole screen). Also on line 2519.

Reply:
Given that this is true any time we call [...textView setNeedsDisplayOnColumn...] I have placed the check there so that most parts of the code don't need to be concerned with this implementation detail (ie the inefficiency of setNeedsDisplayOnCol), but all benefit from this check.